### DISCUSSION TITLE
Related to smart contract

### DISCUSSION THREAD

**Discussion by dev-Crypto360** [Cyfrin/foundry-full-course-cu#1153] [Q&A]
fix all grammar problem

text:
"how can I create a smart contract without using open-plan ready code? how do I write all the code on remix? like a pro solidity dev, without using anyone else code.
I will be very helpful if anyone replies to my answer related to my question ."

**Comment by Merulez** [comment]
Well ,I think it is obvious that you have to learn how to code and you should be first a solidity developer and know the fundamentals of developing a smart contract and it is a smart practice to use and inherit already proven code with no bugs in it into your protocol / code.
